webpack/assets/javascripts/react_app/components/charts/Chart.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Chart = ({ cssClass, id, hasData }) => {
+  return hasData ?
+    <div className={cssClass} id={id + 'Chart'}></div> :
+    (
+      <div className="alert alert-warning">
+        <span className="pficon pficon-warning-triangle-o"></span>
+        <strong>{_('No data').toString()}</strong>.
+      </div>
+    );
+};
```

Comment:
nitpick for the future, this should be refactored to be less coupled and use as a stand alone component (without statistics usage case).